pkg/lib/cockpit-components-listing.jsx

```diff
@@ -241,12 +231,12 @@ var ListingRow = React.createClass({
             }
 
             return (
-                <tbody className="open">
+                <tbody key={this.props.rowId} className="open">
```

Comment:
Let's not add keys to top level components, they should be added when instantiating a list of children.